library/ecp_curves.c

```diff
@@ -580,6 +581,41 @@ static inline void ecp_mpi_load( mbedtls_mpi *X, const mbedtls_mpi_uint *p, size
     X->p = (mbedtls_mpi_uint *) p;
 }
 
+/*
+ * Domain parameters for SM2 (GM/T 0003 Part 5)
```

Comment:
Please put these together with the others, i.e. just before the definition of `ECP_LOAD_GROUP`.

Reply:
OK